### PR TITLE
CEDS-2124 - initial implementation

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -159,3 +159,8 @@ whitelist {
   enabled = false
   eori = []
 }
+
+timeoutDialog {
+  timeout="13min"
+  countdown="2min"
```
Comment (MaciejRewera) [inline conf/application.conf:165]: Is it working in this form? Thought it has to be sth like `13 MINUTES`. Depends on the parser I guess...

Comment (SteveSugden) [reply conf/application.conf:165]: Yes - seems to work.  I've been testing with 
```
timeoutDialog {
  timeout="130s"
  countdown="2min"
}
```

File: app/assets/stylesheets/customsdecexfrontend-app.scss [hunk 2]
```diff
@@ -21,4 +21,5 @@ $govuk-fonts-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/ass
 
 //Below are all the CSS Overwrites that are required in order for the UI to align with the prototype
 @import 'partials/_gdsSummaryLists.scss';
-@import 'partials/_gdsBase.scss';
\ No newline at end of file
+@import 'partials/_gdsBase.scss';
+@import 'partials/_timeout-dialog.scss';
```
Comment (PauloPortugal) [inline app/assets/stylesheets/customsdecexfrontend-app.scss:25]: :+1:

File: app/controllers/SessionTimeoutController.scala [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future.successful
+
+class SessionTimeoutController @Inject()(authenticate: AuthAction, mcc: MessagesControllerComponents, sessionTimedOut: session_timed_out)(
+  implicit ec: ExecutionContext
+) extends FrontendController(mcc) with I18nSupport {
+
+  def signOut(): Action[AnyContent] = authenticate.async { implicit request =>
+    successful(Results.Redirect(routes.SessionTimeoutController.signedOut()).withNewSession)
+  }
```
Comment (PatrykRudnicki) [inline app/controllers/SessionTimeoutController.scala:35]: You can remove `.async` and `successful`.

Async is only necessary when you have a Future. Here you don't have so this is not necessary, like you have in method below.

**Pull request merged**